.prettierrc

```diff
@@ -1,3 +1,5 @@
 {
-  "singleQuote": true
+  "singleQuote": true,
+  "printWidth": 80,
```

Comment:
Added because I initially saw a huge diff.

Reply:
I've been seeing the same :thinking:

Reply:
Hopefully this will prevent future issues. It could be that I had some higher-level Prettier config floating around somewhere.